backend/src/main/java/ai/verta/modeldb/versioning/blob/diffFactory/DatasetBlobDiffFactory.java

```diff
@@ -2,14 +2,8 @@
 
 import static ai.verta.modeldb.versioning.blob.diffFactory.ConfigBlobDiffFactory.removeCommon;
 
-import ai.verta.modeldb.versioning.BlobDiff;
-import ai.verta.modeldb.versioning.BlobExpanded;
-import ai.verta.modeldb.versioning.DatasetBlob;
-import ai.verta.modeldb.versioning.DatasetDiff;
-import ai.verta.modeldb.versioning.PathDatasetComponentBlob;
-import ai.verta.modeldb.versioning.PathDatasetDiff;
-import ai.verta.modeldb.versioning.S3DatasetComponentBlob;
-import ai.verta.modeldb.versioning.S3DatasetDiff;
+import ai.verta.modeldb.versioning.*;
```

Comment:
@conradoverta : are these changes expected?

Reply:
Yes. I was just adding an example and the autoformatting did this I think.